configure.ac

```diff
@@ -18,6 +18,7 @@ AC_CANONICAL_HOST   # host_cpu, host_vendor, and host_os
 AM_INIT_AUTOMAKE([foreign subdir-objects 1.9 tar-pax])
 AM_MAINTAINER_MODE
 AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_MACRO_DIRS([m4])
```

Comment:
Was this an intentional change? As far as I know, where it is supported it should have identical behaviour to the following line.